docs/sphinx/Contact.rst

```diff
@@ -0,0 +1,18 @@
+Contact
+=======
+
+For any questions or suggestions please contact us at: chipsec@intel.com
+
+We also have the `issue tracker <https://github.com/chipsec/chipsec/issues>`_ in our GitHub repo. If you'd like to report a bug or make a request please open an issue.
+
+If you'd like to make a contribution to the code please open a `pull request <https://github.com/chipsec/chipsec/pulls>`_
+
+Mailing lists:
+
+ * CHIPSEC users: https://groups.google.com/forum/#!forum/chipsec-users
```

Comment:
Just checking - I can't access this address. Is it for subscription only?

Reply:
The group is there, but the permissions got messed up and we need @johnloucaides to help us fix it as he's the owner.